src/__mocks__/handlers/ResourceEditor/handlers.ts

```diff
@@ -0,0 +1,358 @@
+            'https://bbp.epfl.ch/nexus/v1/files/public/sscx/15b402f7-c94e-47d7-8678-86914a54fcb1',
+        },
+        {
+          contentUrl:
+            'https://bbp.epfl.ch/nexus/v1/files/public/sscx/3aed62da-c4cb-42bf-9f01-691c7f22b151',
+        },
+      ],
+      name: 'tkb060126a2_ch3_bc_n_jh_100x_1',
+    },
+    {
+      '@id':
+        'https://bbp.epfl.ch/neurosciencegraph/data/neuronmorphologies/3c580d32-2e76-47db-a340-ff09b0fd0199',
+      '@type': ['NeuronMorphology', 'Entity'],
+      distribution: [
+        {
+          contentUrl:
+            'https://bbp.epfl.ch/nexus/v1/files/public/sscx/e2868f5d-a789-4b73-9fc5-296996c1eb68',
+        },
+        {
+          contentUrl:
+            'https://bbp.epfl.ch/nexus/v1/files/public/sscx/9b09a6ce-09eb-430d-9a62-769815460257',
+        },
+      ],
+      name: 'rp100428-12_idK',
+    },
+    {
+      '@id':
+        'https://bbp.epfl.ch/neurosciencegraph/data/neuronmorphologies/70e1112d-3889-4e80-9333-c09cf1c252d4',
+      '@type': ['Entity', 'NeuronMorphology'],
+      distribution: [
+        {
+          contentUrl:
+            'https://bbp.epfl.ch/nexus/v1/files/public/sscx/b51547a1-d341-409d-ace5-7e28e12d26d4',
+        },
+        {
+          contentUrl:
+            'https://bbp.epfl.ch/nexus/v1/files/public/sscx/8e0332e0-b89c-4690-8998-d69ce4cdaea8',
+        },
+      ],
+      name: 'rp110119_L5-2_idC',
+    },
+  ],
+  license: {
+    '@id':
+      'https://bbp.epfl.ch/neurosciencegraph/data/licenses/97521f71-605d-4f42-8f1b-c37e742a30bf',
+    '@type': 'License',
+  },
+  name: 'In vitro-filled neuron morphologies from layer 6 with m-type L6_SBC',
+  objectOfStudy: {
+    '@id':
+      'http://bbp.epfl.ch/neurosciencegraph/taxonomies/objectsofstudy/singlecells',
+    '@type': 'ObjectOfStudy',
+    label: 'Single Cell',
+  },
+  subject: {
+    '@type': 'Subject',
+    species: {
+      '@id': 'http://purl.obolibrary.org/obo/NCBITaxon_10116',
+      label: 'Rattus norvegicus',
+    },
+  },
+  _constrainedBy: 'https://neuroshapes.org/dash/dataset',
+  _createdAt: '2020-03-09T09:03:23.680Z',
+  _createdBy: 'https://bbp.epfl.ch/nexus/v1/realms/bbp/users/akkaufma',
+  _deprecated: true,
+  _incoming:
+    'https://bbp.epfl.ch/nexus/v1/resources/public/sscx/datashapes:dataset/neuronmorphologies%2F23d3d87e-94fe-4639-b5c8-a26a712587e6/incoming',
+  _outgoing:
+    'https://bbp.epfl.ch/nexus/v1/resources/public/sscx/datashapes:dataset/neuronmorphologies%2F23d3d87e-94fe-4639-b5c8-a26a712587e6/outgoing',
+  _project: 'https://bbp.epfl.ch/nexus/v1/projects/public/sscx',
+  _rev: 9,
+  _schemaProject: 'https://bbp.epfl.ch/nexus/v1/projects/public/sscx',
+  _self:
+    'https://bbp.epfl.ch/nexus/v1/resources/public/sscx/datashapes:dataset/neuronmorphologies%2F23d3d87e-94fe-4639-b5c8-a26a712587e6',
+  _updatedAt: '2021-11-04T11:45:59.645Z',
+  _updatedBy: 'https://bbp.epfl.ch/nexus/v1/realms/bbp/users/arnaudon',
+};
+
+const getResolverResponseForobjectOfStudy = rest.get(
```

Comment:
small thing = the `o` in `object` in the name of the function should be capital if we are going with camel case. So `getResolverResponseForObjectOfStudy`